fe/fe-core/src/main/java/org/apache/doris/catalog/Replica.java

```diff
@@ -46,10 +46,12 @@
         SCHEMA_CHANGE,
         CLONE,
         ALTER, // replica is under rollup or schema change
-        DECOMMISSION; // replica is ready to be deleted
+        DECOMMISSION, // replica is ready to be deleted
+        COMPACTION_TOO_SLOW; // replica version count is too large
+
 
         public boolean canLoad() {
-            return this == NORMAL || this == SCHEMA_CHANGE || this == ALTER;
+            return this == NORMAL || this == SCHEMA_CHANGE || this == ALTER || this == COMPACTION_TOO_SLOW;
```

Comment:
This is meaningless.
Because every time you set the state to `COMPACTION_TOO_SLOW`, you also set replica `isBad` to `true`.
I think we can only set replica's state to `COMPACTION_TOO_SLOW`, and change logic in `replica.isAlive()`.